sdk/python/tests/integration/feature_repos/repo_configuration.py

```diff
@@ -46,6 +46,17 @@ class IntegrationTestRepoConfig:
     infer_event_timestamp_col: bool = True
     infer_features: bool = False
 
+    def __repr__(self) -> str:
```

Comment:
What are you doing here?

Reply:
Making the output of pytest easier to read - it produces something like:
```
test_online_retrieval[Provider: local-Fil-sqlite]
```

instead of something like:
```
test_online_retrieval[IntegrationTestRepoConfig(provider='gcp', online_store='datastore', offline_store_creator=<class 'tests.integration.feature_repos.universal.data_sources.bigquery.BigQueryDataSourceCreator'>, full_feature_names=True, infer_event_timestamp_col=True, infer_features=False)]
```